test/unity/unity_toolkit.cxx

```diff
@@ -10,7 +10,8 @@
 #include <unity/lib/variant_converter.hpp>
 #include <unity/lib/simple_model.hpp>
 #include <unity/lib/unity_sarray.hpp>
-#include <unity/lib/api/model_interface.hpp>
+// #include <unity/lib/api/model_interface.hpp>
```

Comment:
Why is this include commented out, with no other change in this file?

Reply:
If it isn't needed for compilation, let's remove it entirely (instead of commenting it out). It looks like maybe a change incompatible with this branch crept in somewhere.

Reply:
Done!